tools/autograd/templates/variable_factories.h

```diff
@@ -143,8 +143,8 @@ AT_FORALL_SCALAR_TYPES_AND3(Bool, Half, BFloat16, TENSOR)
     InitListTensorType type_;
   };
 
-  inline std::ostream& operator<<(std::ostream& stream, const InitListTensor& list_init_tensor) {
-    list_init_tensor.pretty_print_recursive(stream);
+  inline std::ostream& operator<<(std::ostream& stream, const InitListTensor& init_list_tensor) {
+    init_list_tensor.pretty_print_recursive(stream);
     return stream;
   }
 } // namespace detail
```

Comment:
Regarding line 51 `TORCH_CHECK(init_list.size() > 0, "Empty init-list is not supported");`, maybe it would be good to write "Empty init-list is not yet supported. Here are some alternatives: ..."